README.md

```diff
@@ -405,7 +405,16 @@ amadeus.shopping.hotelOffersSearch.get({
 // Check offer conditions of a specific offer id
 amadeus.shopping.hotelOfferSearch('XXX').get()
 
-// Hotel Booking API
+// Hotel Booking API V2
```

Comment:
Let's write it v2 instead of V2 so it will be consistent with the [changelog](https://github.com/amadeus4dev/amadeus-node/blob/master/CHANGELOG.md) of the library.